src/sections/Meshmap/Meshmap-collaborate/collaboration-feature-work.js

```diff
@@ -39,7 +39,7 @@ const CollaborationFeatureWrapper = styled.div`
 
     h2 {
       span {
-        color: #00b39f;
+        color: #FFFFFF;
```

Comment:
Use theme referencing to reference the color

Reply:
I've chosen colors from the Layer5 design itself.